ios/browser/application_context/BUILD.gn

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//build/config/ios/rules.gni")
+import("//ios/build/config.gni")
+
+source_set("application_context") {
+  sources = [
+    "brave_application_context_impl.h",
+    "brave_application_context_impl.mm",
+  ]
+  deps = [
+    "//base",
+    "//brave/components/brave_component_updater/browser",
+    "//brave/components/https_upgrade_exceptions/browser",
```

Comment:
why is this here?

Reply:
It must have been a mistake when I was trying to expose this too. Eventually I'll have to do this too. I can remove it for now

Reply:
Removed